lisa/sut_orchestrator/azure/features.py

```diff
@@ -67,3 +69,14 @@ def _get_console_log(self, saved_path: Optional[Path]) -> bytes:
         log_response = requests.get(diagnostic_data.serial_console_log_blob_uri)
 
         return log_response.content
+
+
+class Gpu(AzureFeatureMixin, features.Gpu):
+    def _initialize(self, *args: Any, **kwargs: Any) -> None:
+        super()._initialize(*args, **kwargs)
+        self._initialize_information(self._node)
+
+    def _is_supported(self) -> None:
```

Comment:
1. If the method is not called only in this class, remove the underscore.
2. the name `is_` hints it returns a bool. Because it raise exception, it should be like `check_supported`.

Reply:
Fixed